docs/useful_tools.md

```diff
@@ -107,6 +110,7 @@ Description of all arguments
 
 - `config`: The path of a model config file.
 - `model`: The path of a ONNX model file.
```

Comment:
Here could be updated since that both onnx file and TensorRT engine file could be the input.